src/iexamine.cpp

```diff
@@ -3431,6 +3432,144 @@ void iexamine::climb_down( player &p, const tripoint &examp )
+                        _("Light: Eat when hungry. Pay some attention to food use."),
+                        _("Medium: Eat when hungry. Always cook nutritious food."),
+                        _("Heavy: Eat when very hungry. Optimize food usage."), NULL);
+                    p.home.set_food_ration(lv-1);
+                }
+                case 2: {
+                    int lv = menu(true, _("Ration Level:"),
+                        _("None: Use anything and everything you want."),
+                        _("Light:"),
+                        _("Medium:"),
+                        _("Heavy:"), NULL);
+                    p.home.set_ammo_ration(lv-1);
+                }
+                case 3: {
+                    int lv = menu(true, _("Ration Level:"),
+                        _("None: Use anything and everything you want."),
+                        _("Light:"),
+                        _("Medium:"),
+                        _("Heavy:"), NULL);
+                    p.home.set_med_ration(lv-1);
+                }
+                case 4: {
+                    goto top_menu;
+                }
+            }
+            goto sub_menu;
+        }
+        case 2: {//ammo type prefrences
+            add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+            //TODO: Menu assign prefrence (weight) to ammo types.  Prefrence to be refrenced by NPC choosing weapon.
+        }
+        case 3: {//crafting priorities
+            add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+            //TODO: Menu assign priorities (weight) to craft recipies.  NPCs to refrence priorities when crafting.
+        }
+        case 4: {//view status.
+            iexamine::base_view_ele();
+        }
+        case 5: {//base planner
+            add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+        }
+        case 6: {//personel managment
+            add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+        }
+    }
+}
+
+/**
+ * View base status/information from a bulletin board. (limited info)
+ */
+void iexamine::base_view( player &p, const tripoint &examp) {
+    add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+}
+
+/**
+ * View base status/information from a monator. (lots of info)
+ */
+void iexamine::base_view_ele( player &p, const tripoint &examp) {
+    add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+}
+
+/**
+ * Manually disconnect random remote system(s).  With a hammer or wrench you can fix the connection.
+ * LOLz joke method. 
+ */
+void iexamine::base_discon( player &p, const tripoint &examp ) {
+    add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+}
+
+/**
+ * Mannage basic and security remote systems.  (Lights! Cammera! Action...of connected turrets!) 
+ */
+void iexamine::base_sec( player &p, const tripoint &examp ) {
+    add_msg("Not implemented yet.","(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
```

Comment:
What are those additional arguments supposed to be? They are not used at all (there is no "%s" in the message).

Reply:
It;s someone flipping a table in frustration.  It's just supposed to be another line and it won't be staying anyways.

Reply:
>  It's just supposed to be another line

If you want another message line, either call `add_msg` again (each call generates one line), or add a line break to the message: `add_msg("Line 1\nLine 2")`.

And another thing: string literals that should be translated must be surrounded by `_(` and `)` - see examples all over the code.

Reply:
I noticed the `_( )` thing and started putting it in without knowing the meaning.  The line is just a placeholder until I actually write the method since an empty method looks weird to me.